source/multiple-mongosyncs.txt

```diff
@@ -0,0 +1,260 @@
+.. _c2c-sharded-clusters:
+
+=====================================
+Use ``mongosync`` on Sharded Clusters
+=====================================
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+There are two ways to synchronize :ref:`sharded clusters
+<sharded-cluster>`. You can use either one ``monogosync`` or several
+``monogosync`` instances. For best performance, use multiple
+``monogosync`` instances, one ``monogosync`` for each shard in the
+cluster.
+
+.. _c2c-sharded-config-single:
+
+Configure a Single ``mongosync`` Instance
```

Comment:
@huayu-ouyang  I'm not sure what the use cases are for running a single mongosync. Is it better to demote the single monogsync method to a note or should it be included along with the multiple monogsync method?

Reply:
Thanks @ldennis i've modified the text to read: 

For best performance with large or heavily loaded clusters, use multiple ``monogosync`` instances,

And I'll leave the single monogsync case as is.